hasher-matcher-actioner/hmalib/lambdas/api/matches.py

```diff
@@ -133,21 +136,20 @@ def get_match_details(table: Table, content_id: str) -> t.List[MatchDetail]:
 def get_signal_details(
     table: Table, signal_id: t.Union[str, int], signal_source: str
 ) -> t.List[MatchDetailMetadata]:
+    # TODO: Remove need for signal_source. That's part of the *SignalMetadata class now.
     if not signal_id or not signal_source:
         return []
 
     return [
         MatchDetailMetadata(
-            dataset=metadata.ds_id,
+            dataset=metadata.privacy_group_id,
```

Comment:
nit: finish the job and update `MatchDetailMetadata` dataset to privacy_group as well.